src/main/java/org/cascadebot/cascadebot/commands/moderation/ModlogChannelSubCommand.java

```diff
@@ -0,0 +1,199 @@
+            context.getUiMessaging().replyUsage();
+            return;
+        }
+        TextChannel channel = DiscordUtils.getTextChannel(context.getGuild(), context.getArg(1));
+        if (channel == null) {
+            context.getTypedMessaging().replyDanger("Couldn't find channel `" + context.getArg(1) + "`");
+            return;
+        }
+        if (!context.getData().getModeration().getModlogEvents().containsKey(channel.getIdLong())) {
+            context.getTypedMessaging().replyDanger("The specified channel is not a modlog channel!");
+            return;
+        }
+        if (context.getArg(0).equalsIgnoreCase("enable")) {
+            if (isEnabled(channel, context)) {
+                context.getTypedMessaging().replyDanger("Channel is already enabled!");
+                return;
+            }
+            channel.createWebhook("cascade-modlog").queue(webhook -> {
+                context.getData().getModeration().getModlogEvents().get(channel.getIdLong()).setNewWebhook(webhook.getIdLong(), webhook.getToken());
+                context.getTypedMessaging().replySuccess("Successfully disabled channel!");
+            }, throwable -> {
+                context.getTypedMessaging().replyDanger("Failed to enable events! The bot probably doesn't have permissions to create webhooks.");
+            });
+        } else if (context.getArg(0).equalsIgnoreCase("disable")) {
+            if (!isEnabled(channel, context)) {
+                context.getTypedMessaging().replyDanger("Channel is already disabled!");
+                return;
+            }
+            channel.deleteWebhookById(String.valueOf(context.getData().getModeration().getModlogEvents().get(channel.getIdLong()).getWebhookId())).queue(aVoid -> {
+                context.getTypedMessaging().replySuccess("Successfully enabled channel!");
+            }, throwable -> {
+                context.getTypedMessaging().replyDanger("Failed to disable events! The bot probably doesn't have permissions to delete webhooks.");
+            });
+        } else if (context.getArg(0).equalsIgnoreCase("delete")) {
+            context.getData().getModeration().getModlogEvents().remove(channel.getIdLong());
+            context.getTypedMessaging().replySuccess("Successfully deleted channel!");
+        } else if (context.getArg(0).equalsIgnoreCase("info")) {
+            GuildSettingsModeration.ChannelModlogEventsInfo channelModlogEventsInfo = context.getData().getModeration().getModlogEvents().get(channel.getIdLong());
+            String events = channelModlogEventsInfo.getEvents().stream().map(event -> Language.i18n(context.getLocale(), "enums.moldogevent." + event.name().toLowerCase() + ".display")).collect(Collectors.joining("\n"));
+            context.getUiMessaging().sendPagedMessage(PageUtils.splitStringToEmbedPages(events, "Events for channel " + channel.getName(), 1000, '\n'));
+        } else {
+            context.getUiMessaging().replyUsage();
+        }
```

Comment:
There are a lot of if statements here, a switch statement that called other methods would be more appropriate I feel

Reply:
I'll try to convert as much as I can, but a lot of these if states are because of making sure args length are correct. Also I'm just using `equalsIgnoreCase` for testing. I need to switch it over to whatever our proper arg checker is.

Speaking of that we need to go through all the commands at some point and switch everything over to our proper arg check so language works properly.

Reply:
Agreed yeah, we do need to do that